documentation/modules/rn-2.5.adoc

```diff
@@ -36,6 +36,12 @@ The user interface of create and update providers now aligns with the look and f
 
 The old UI of MTV 2.3 cannot be enabled by setting `feature_ui: true` in ForkliftController anymore.
 
+.Errors logged in populator pods are improved
+
+// Need help with this
+In {project-short}, the populator pod redirects the `ovirt-img` errors to the `/dev/termination-log`, so that populator controller can pick it up from  `pod.Status.ContainerStatuses[0].State.Terminated.Message`.
+// https://issues.redhat.com/browse/MTV-818 -
```

Comment:
@liranr23 & @ahadas - please can you advise what else you like included about [MTV-818](https://issues.redhat.com/browse/MTV-818)

Reply:
The populator controller will monitor the populator pod and recreate the populator pod in case of an error 3 times. On the third time, the populator pod will remain in fail status. This allows the user to pick up the error from the populator pod and let the forklift-controller to know this step has failed.

Reply:
@liranr23 - ta very much